projects/aca-testing-shared/src/components/viewer/viewer.ts

```diff
@@ -93,4 +94,24 @@ export class Viewer extends Component {
     const count = await this.pdfViewerContentPages.count();
     return count > 0;
   }
+
+  async expectUrlToContain(text: string): Promise<void> {
```

Comment:
This should be somewhere in a parent class.... or nowhere....
It only has one line of code, no need to have a separate method for it, just write the expect in the test